### DISCUSSION TITLE
No picture from rear camera on RLink(( Please help the experts)

### DISCUSSION THREAD

**Discussion by StepanenkoNadejda** [cedricp/ddt4all#1140] [Q&A]
Hello,
Who faced the situation. I connected the rear-view camera to the RLINK (yellow and black wires) Megane 3, activated it via ddt4all, the camera is connected to the rear lights. The camera turns on, but there is no picture on the screen. I've heard what additional configurations need to be installed. It's possible?

**Comment by AndyIreland** [comment]
have you wired up the composite video in? :

![image](https://user-images.githubusercontent.com/49172217/196780007-93cbd7c4-6de0-4c49-9908-65d9efd00d83.png)

![image](https://user-images.githubusercontent.com/49172217/196782407-d9c4e832-8804-4957-8eb3-4c56d13182b9.png)

**Comment by AndyIreland** [comment]
Did you reboot the rlink after making your changes?

**Comment by StepanenkoNadejda** [comment]
> Did you reboot the rlink after making your changes?

I did everything as written. I connected the camera RCA connection to 18 (yellow) and 21 (black), only not in the RLink but under the driver's seat. Configured for DDT4all, rebooted R Link, and nothing(((

**Comment by StepanenkoNadejda** [comment]
The camera is working, the backlight turns on when I turn on the reverse gear. Do I need any additional settings, for example: parking sensors, maybe I need to change the screen resolution somehow?

**Comment by AndyIreland** [comment]
I wonder if the the wires under the seat are connected up to the back of the radio - in any case i thought the wire for the reverse camera were up  in the footwell area , not under the seat? 

there is also a CVBS (video in) ground as well on pin 19

**Comment by StepanenkoNadejda** [comment]
> I wonder if the the wires under the seat are connected up to the back of the radio - in any case i thought the wire for the reverse camera were up in the footwell area , not under the seat?
> 
> there is also a CVBS (video in) ground as well on pin 19

Yes. 19 and 21, they are the same turns out (grounding).

**Comment by StepanenkoNadejda** [comment]
![YbaH-w0_wfzVVlhqQfhRxSLj-xY-1920](https://user-images.githubusercontent.com/36893353/196790977-43f5dc45-2a5e-45ef-aec3-31c220febc50.jpg)
![8K73txpu-hhz9BPmxEnndNH8blI-1920](https://user-images.githubusercontent.com/36893353/196790984-04b7c13b-2ba0-4791-9b27-f4d11e1bd5e8.jpg)

**Comment by AndyIreland** [comment]
if that were me i would pull off the trim , take out the rlink screen and put the wires straight to the back of the multi connector and see what happens. - its a good sign the screen goes black but backlit when you put car in reverse .
Can you see any reversing guidelines on the screen?

**Comment by StepanenkoNadejda** [comment]
> if that were me i would pull off the trim , take out the rlink screen and put the wires straight to the back of the multi connector and see what happens. - its a good sign the screen goes black but backlit when you put car in reverse . Can you see any reversing guidelines on the screen?

I also connected the R Link to the screen. Nothing. When the reverse speed is turned on, a black screen with guide lines appears, but there is no picture.

**Comment by AndyIreland** [comment]
and you are absolutely sure the camera is working - if the camera is 12v , try putting the positive up to a constant positive feed and not on the reverse light bulb socket

**Comment by StepanenkoNadejda** [comment]
> and you are absolutely sure the camera is working - if the camera is 12v , try putting the positive up to a constant positive feed and not on the reverse light bulb socket

Thank you! I'll try for the sake of experiment, maybe something will change. But as the camera has LEDs, they will be constantly lit as soon as the car will turn on.

**Comment by AndyIreland** [comment]
> > and you are absolutely sure the camera is working - if the camera is 12v , try putting the positive up to a constant positive feed and not on the reverse light bulb socket
> 
> Thank you! I'll try for the sake of experiment, maybe something will change. But as the camera has LEDs, they will be constantly lit as soon as the car will turn on.

are they infra red LED's though - the ones that help the camera see in the dark? - if they are you cannot see them lit up just with eye, but if you point your phone camera at them you will see them lit up

**Comment by StepanenkoNadejda** [comment]
![20221020_084523](https://user-images.githubusercontent.com/36893353/196875711-7b7a837c-58c3-4e48-836f-63ec6402b029.jpg)

**Comment by AndyIreland** [comment]
so , when you have it wired up to the reverse bulbs when you put the car into reverse the LED lights are lit? - or do they only light up when its dark outside?

**Comment by StepanenkoNadejda** [comment]
> > > and you are absolutely sure the camera is working - if the camera is 12v , try putting the positive up to a constant positive feed and not on the reverse light bulb socket
> > 
> > 
> > Thank you! I'll try for the sake of experiment, maybe something will change. But as the camera has LEDs, they will be constantly lit as soon as the car will turn on.
> 
> are they infra red LED's though - the ones that help the camera see in the dark? - if they are you cannot see them lit up just with eye, but if you point your phone camera at them you will see them lit up

No, I have a camera with LED's lighting, not infrared. Does it matter if the camera is with an ССD matrix or without an ССD?

**Comment by StepanenkoNadejda** [comment]
> so , when you have it wired up to the reverse bulbs when you put the car into reverse the LED lights are lit? - or do they only light up when its dark outside?

LED is constantly glowing when I turn on the reverse gear. It doesn't matter day or night, since there is no light sensor on the camera. I also tried to connect the other 2 cameras. Nothing(

**Comment by StepanenkoNadejda** [comment]
> > so , when you have it wired up to the reverse bulbs when you put the car into reverse the LED lights are lit? - or do they only light up when its dark outside?
> 
> LED is constantly glowing when I turn on the reverse gear. It doesn't matter day or night, since there is no light sensor on the camera. I also tried to connect the other 2 cameras. Nothing(

They also write that you need to configure: multimedia, a parking sensor unit — so that the car understands what we want from it. To do this, the record of the type of your parking sensors (standard for example RDAVAR, RDPRAR, SRDPRO) should simply be replaced with RDAR01.

**Comment by StepanenkoNadejda** [comment]
> ![image](https://user-images.githubusercontent.com/36893353/196994293-9d229678-0025-45cd-8b9f-6d4e52044563.png)

But I don't know how do it?)

**Comment by AndyIreland** [comment]
> > so , when you have it wired up to the reverse bulbs when you put the car into reverse the LED lights are lit? - or do they only light up when its dark outside?
> 
> LED is constantly glowing when I turn on the reverse gear. It doesn't matter day or night, since there is no light sensor on the camera. I also tried to connect the other 2 cameras. Nothing(

Can you make a screenshot of your settings for these parameters:

![image](https://user-images.githubusercontent.com/49172217/197040283-728b810f-c508-47ee-886d-0302da6a7cb3.png)

**Comment by AndyIreland** [comment]
and a screenshot of yours for this:

![image](https://user-images.githubusercontent.com/49172217/197041450-2b1d7b83-b467-4894-9e17-d9f72a2e0340.png)

**Comment by StepanenkoNadejda** [comment]
> and a screenshot of yours for this:
> 
> ![image](https://user-images.githubusercontent.com/49172217/197041450-2b1d7b83-b467-4894-9e17-d9f72a2e0340.png)

Yes, of course)

![2022-10-21_08-43-29](https://user-images.githubusercontent.com/36893353/197135454-0644d292-930f-44b1-948c-16d131175e56.png)
![2022-10-21_08-44-34](https://user-images.githubusercontent.com/36893353/197135459-5d0ae524-e545-4c39-a972-ab34697e8875.png)

**Comment by AndyIreland** [comment]
thanks for that , looks all ok - have you got reverse parking sensors on your car? 

Also where it says trunk handler message set that to not fitted and  the trunk open handler disable rvc set that to not fitted

**Comment by StepanenkoNadejda** [comment]
> thanks for that , looks all ok - have you got reverse parking sensors on your car?
> 
> Also where it says trunk handler message set that to not fitted and the trunk open handler disable rvc set that to not fitted

Yes, I have rear parking sensor).

![2022-10-21_08-44-34 — копия](https://user-images.githubusercontent.com/36893353/197247831-f4a17454-56fb-4597-8474-0ab3ec5a0ea6.png)

**Comment by StepanenkoNadejda** [comment]
> thanks for that , looks all ok - have you got reverse parking sensors on your car?
> 
> Also where it says trunk handler message set that to not fitted and the trunk open handler disable rvc set that to not fitted

can there be a decoder in the car that does not allow the image to be displayed on the screen? 3 cameras tried to connect. Nothing(

**Comment by BarektE** [comment]
> ![20221020_084523](https://user-images.githubusercontent.com/36893353/196875711-7b7a837c-58c3-4e48-836f-63ec6402b029.jpg)

This camera does not work on R-Link.

**Comment by AndyIreland** [comment]
> >
> 
> This camera does not work on R-Link.

why not? - are you giving it a proper 12v power supply and not the 6v supply from the rlink?

**Comment by Furtif** [comment]
good post need a ref .. but clean in ... please refer it at discutions 
https://github.com/cedricp/ddt4all/issues/836

**Comment by Dorinmomo** [comment]
I have the same problem with Rlink 1, no image from rear cam. Did anyone fixed it? Can you tel me how?

**Comment by Dorinmomo** [comment]
I'm back !! I did it !!! OMG !! I have image on my rear camera. R link 1 renault captur. Activated it with ddt4all , bought a chinese connection cable  that said "for renault" but it did not work.... I thought it was the camera, tested all the wires, 👎 .... The thing was that the wires where in the wrong place(pins). I did not check if it was on the 18, 19 pins. Took all out and reconect the wires and it worked. The camera was aftermarket, like in the pictures. 
![17159652722611039177459834047726](https://github.com/cedricp/ddt4all/assets/170031235/2ada513b-92e0-407a-9436-0dea787225c6)
Thank you for posting here .

**Comment by Dorinmomo** [comment]
> I wonder if the the wires under the seat are connected up to the back of the radio - in any case i thought the wire for the reverse camera were up in the footwell area , not under the seat?
> 
> there is also a CVBS (video in) ground as well on pin 19

Probably the wires are not up to the screen, mine had not. You need to go with the wires up to the back of he screen. That's why you do not have image. 
![IMG_20240517_172346](https://github.com/cedricp/ddt4all/assets/170031235/e161d97b-a414-4339-81f6-3ac8b7227687)
![IMG_20240517_174627](https://github.com/cedricp/ddt4all/assets/170031235/d12e404c-7b2b-4527-80f8-94b004ed430a)
![IMG_20240517_171538](https://github.com/cedricp/ddt4all/assets/170031235/a2c142aa-d5a6-4c80-ba03-46c451e48c24)
![IMG_20240517_171506](https://github.com/cedricp/ddt4all/assets/170031235/4b979ea7-7ca8-491d-ae78-7d630eaead95)
![IMG_20240517_175445](https://github.com/cedricp/ddt4all/assets/170031235/ef45f5ce-4856-46d0-bee7-1092bd724ef0)

**Comment by AndyIreland** [comment]
if any help to anyone here is wiring diagram to the back of the MFD (multi function touchscreen display) 
![image](https://github.com/cedricp/ddt4all/assets/49172217/fedb80f9-4214-4d6c-94b9-2f5ddccc45d3)

**Comment by R-3MY** [comment]
> I'm back !! I did it !!! OMG !! I have image on my rear camera. R link 1 renault captur. Activated it with ddt4all , bought a chinese connection cable that said "for renault" but it did not work.... I thought it was the camera, tested all the wires, 👎 .... The thing was that the wires where in the wrong place(pins). I did not check if it was on the 18, 19 pins. Took all out and reconect the wires and it worked. The camera was aftermarket, like in the pictures. ![17159652722611039177459834047726](https://private-user-images.githubusercontent.com/170031235/331677223-2ada513b-92e0-407a-9436-0dea787225c6.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc5Mjg2MTgsIm5iZiI6MTcxNzkyODMxOCwicGF0aCI6Ii8xNzAwMzEyMzUvMzMxNjc3MjIzLTJhZGE1MTNiLTkyZTAtNDA3YS05NDM2LTBkZWE3ODcyMjVjNi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYwOVQxMDE4MzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hY2I2ZjZiMzI5OWRkNDhiMDkyZjE1MGM0ZTQ2YWVjYzljNjQzMjQ1ZWQ1MTJlZjAxOWNiNjM4ZjM2ZDliMjljJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.R2KvuBtsqQt3yxsCRmI-2N9O5nO7ir5_C7keNNLh0Gg) Thank you for posting here .

Thank you, you literally saved me! I got the same issue with a 24 pin with wrong pin.

I tested several 12v cameras and I still have the same problem (black screen) however when I put my finger in front of the camera I do have a "display" only it seems to be very noisy. I briefly see guide lines but no display of the image that should come from the camera.

I have a Kangoo II ph 2 with rear sensors.

Regards, Rémy.

**Reply by R-3MY** [reply]
Okay I'm just dumb and cannot count until 20 :/ it put 18 on 19 and 19 on 18 that why i got weird things on screen. now everything work well, i tested 3 cams all working great in 12V.

**Comment by mwoe75** [comment]
Very interesting topic. 
I've just activated android carplay on my car (Megane III 2015) and am now looking to get a reverse cam working.
Car has front and rear parking sensors. When Ik put the car in reverse, the parking sensors are shown on the r-link, with a black screen next to it.

[R-3MY](https://github.com/R-3MY)
Can you share which cameras you bought that work and how you wired it up?

example (which isn't right for instance, because there's no 12v power in the 'factory unit'):
<a href="https://ibb.co/JRFcrKV"><img src="https://i.ibb.co/JRFcrKV/wires.jpg" alt="wires" border="0"></a>

another one:

<a href="https://ibb.co/Wn8fjcr"><img src="https://i.ibb.co/Wn8fjcr/wire2.jpg" alt="wire2" border="0"></a>

**Reply by Dorinmomo** [reply]
Hi there. For the camera you can buy any camera with NTSC, depending where you want to install it. Buy  this 24 pin adapter Just found this amazing item on AliExpress. Check it out! 
RON18.23  27%OFF | 24 Pin Adapter Rear View Camera Connection Cable for Renault&Dacia for Opel for Vauxhall for Clio 4 2012-Up
https://a.aliexpress.com/_ExRFzWF . On it you need to cut the wires of the video signal and put it on the right pins. Check first if you have the same R link version so it wil work properly. Rlink1 . See the post i made. The red wired will not be used. Connect the camera to the + wire of the reverse lamp at the back of the car. Good luck.

**Reply by Dorinmomo** [reply]
The red cable from the "factory unit " will not work. You need to connect the red wire to the reverse light at the back of the car (you may need second person to put the car in reverse to find the +wire. The black wire ( - /GND ), is already connected at the RCA  (19 pin ).

**Reply by R-3MY** [reply]
> Very interesting topic. 
> I've just activated android carplay on my car (Megane III 2015) and am now looking to get a reverse cam working.
> Car has front and rear parking sensors. When Ik put the car in reverse, the parking sensors are shown on the r-link, with a black screen next to it.
> 
> [R-3MY](https://github.com/R-3MY)
> Can you share which cameras you bought that work and how you wired it up?
> 
> example (which isn't right for instance, because there's no 12v power in the 'factory unit'):
> <a href="https://ibb.co/JRFcrKV"><img src="https://i.ibb.co/JRFcrKV/wires.jpg" alt="wires" border="0"></a>
> 
> another one:
> 
> <a href="https://ibb.co/Wn8fjcr"><img src="https://i.ibb.co/Wn8fjcr/wire2.jpg" alt="wire2" border="0"></a>

As @Dorinmomo Say, any NTCS camera is working (i bought 3 cheap camera from Aliexpress and all work fine) but you need to change the 24 pin adapter wire order to put the RCA on pin 18 and 19. (take your time to be 100% sure it's 18 and 19 and in the right order)

**Reply by R-3MY** [reply]
> The red cable from the "factory unit " will not work. You need to connect the red wire to the reverse light at the back of the car (you may need second person to put the car in reverse to find the +wire. The black wire ( - /GND ), is already connected at the RCA  (19 pin ).

I was not using the red wire at all. (i don't have any light on the camera so i think it can be powered on)

![787411017-1268815866.jpg](https://github.com/cedricp/ddt4all/assets/73651704/3ce00c6a-3346-4f6c-b5e5-ce6e6f4bf4e2)

![576644665-24649508.jpg](https://github.com/cedricp/ddt4all/assets/73651704/1dd6af77-bbef-4c84-b300-a3a2e9f36afd)

Btw i was looking for a way to get rear camera while driving forward (i got a Kangoo2 without glass/ middle mirror).
What the purpose of the red wire ? (is it +6V ? for camera) or something for camera "trigger" ?

**Reply by Dorinmomo** [reply]
It is not a trigger for the camera....  I think i have seen something about the image while driving forward,on youtube, i think you need to activate something with ddt4all but i have no other info about it..... Try searching on youtube some tutorials.....

**Reply by R-3MY** [reply]
> It is not a trigger for the camera.... I think i have seen something about the image while driving forward,on youtube, i think you need to activate something with ddt4all but i have no other info about it..... Try searching on youtube some tutorials.....

I found that medianav 24 pin got reverse gear pin (pin 3) :
![1](https://github.com/cedricp/ddt4all/assets/73651704/748244b0-4f1d-4e03-a488-1773dc25d50e)

Okay so did you if rlink1 got this reverse gear pin somewhere ?
I found this youtube video :
https://www.youtube.com/watch?v=olhrPau4AIA

**Reply by Dorinmomo** [reply]
I see that medianav needs the camera to be connected at 21 and 22 pins.... The Rlink 1 does not need a signal from the camera wires to activate the screen on reverse if that is what you are asking, it does that from the software.... I think.... The medianav showld do that to. Try that first....

**Comment by mwoe75** [comment]
So just to be clear: Need to be like this?
<a href="https://ibb.co/SrDK153"><img src="https://i.ibb.co/SrDK153/wiring1.jpg" alt="wiring1" border="0"></a>

Other question::
Which type of cameras do you tried?
I know it needs to be NTSC, but I see resolutions like 720x488 and 1280x720. Also cameras that support both CVBS and AHD format by currting a wire. Does this matter?

**Reply by Dorinmomo** [reply]
Yes, the picture is correct. The resolution does not matter from what i know..... But if you want to see well you need a hi resolution. The part with cutting a wire is for showing the parking aid lines from the camera. Try it without cutting the wire first.....

**Reply by alleenogen** [reply]
camera is working 👍 
Only issue is that the microphone in the car isn't working anymore.
I think the issue is the red wire (pin 3 ) on the white cable adapter.
For Renault megane III with tomtom:
3= Micro supply              Supply for microphone.            5V

Did you removed the white cables manually to rearrange them?
https://fccid.io/ANATEL/00355-14-09666/Manual_1ME09_rev1/F2B1E423-3290-440A-99C1-0BD994D5C828/PDF

Update:
Very easy to remove wires from both connectors.
Just press something small (like a needle) in the holes above the male/female pins and you can get them out.
The red cable on pin 3 is for external power for the cam that I don't use (and pin 3 doesn't give power for my car anyway, but is the micropone).
I removed the cable from pin 24 (not used) and moved it to pin 3. It works now 👍

**Reply by mwoe75** [reply]
One other thing to note about the Cable Adapter: I had to remove the pieces marked in red:
![issue](https://github.com/cedricp/ddt4all/assets/172956321/e1c109af-884e-4201-933d-51220ef21b0e)
The 24p car connector wouldn't fit otherwise.
